app/views/courses/CourseDetailsView.coffee

```diff
@@ -125,6 +129,7 @@ module.exports = class CourseDetailsView extends RootView
     levelSlug = $(e.target).closest('.btn-play-level').data('level-slug')
     levelID = $(e.target).closest('.btn-play-level').data('level-id')
     level = @levels.findWhere({original: levelID})
+    window.tracker?.trackEvent 'Students Class Course Play Level', category: 'Students', courseID: @courseID, courseInstanceID: @courseInstanceID, levelSlug: levelSlug, ['Mixpanel']
```

Comment:
It goes left to right, general to specific.  This one is for the students area, for a course in a class, clicking a play button.

I'm open to suggestions that include all these details somehow.